### DISCUSSION TITLE
Could I change db connection when doing queries using lucid relationships ?

### DISCUSSION THREAD

**Discussion by mortenko** [adonisjs/core#2175] [Help]
Lets say I have two models : Radar and Industry

Radar can be in one industry (1:1) (belongs to)
Industry can be in multiple radars (1:N) (hasMany)

I have default connection pg but I want to use different db connection as you can use by using DatabaseProvider (```Database.connection```)

I want to do something like this:
```
  const radar = await Radar.find(1);
 const industry = await  radar.industry().connection('mysql').fetch()
```

Has anybody tried this ? its possible ?

**Comment by thetutlage** [comment]
Are you using AdonisJS 4.1?

**Reply by mortenko** [reply]
@thetutlage  yes.

**Reply by mortenko** [reply]
unfortunately I dont know typescript and as far as I know testing module was not still migrated.
